templates/webapi/branding/openSUSE/external_reporting.html.ep

```diff
@@ -49,7 +54,15 @@
 %             $version = "12 SP1\x{00A0}(SLED 12 SP1)";
 %         }
 %         # fall back to 'Server' as the most common flavor
-%         $product = join(' ', $flavor_to_prod_sle{$subproduct} // 'Server', $version);
+%         my $sle_product        = $flavor_to_prod_sle{$subproduct} // 'Server';
+%         my $public_sle_product = $public_sle_products{$sle_product};
+%         if ($public_sle_product && ($version =~ qr/(\d+) SP(\d+)/)) {
+%             if ($1 == 15 && $2 >= 3) {
+%                 $distri      = "PUBLIC $distri" ;
+%                 $sle_product = $public_sle_product;
+%             }
+%         }
```

Comment:
I see you have foreseen all possible problems already :) I hope the maintainers of products on bugzilla realize how much mess they cause with their inconsistent naming and changes :(

Reply:
Likely they'll change patterns again for SP4 but then we can still adjust the `>=`.

Reply:
Maybe worth turning into a helper? Logic definitely seems complex enough.

Reply:
Maybe later (one can open a ticket for that). Since this is not a usual template but only part of an optional and distro-specific branding we might not want to mix it into our usual helpers.

Reply:
I agree, this is very site-specific hence in branding. Would be interesting if there could be "site-specific plugin code" elsewhere and then integrated